### PR TITLE
added: authentication page

### PR DESCRIPTION
**Pull request by sharmayatendra**
# Authentication Page

## Authentication contains two pages one is login & another one is signup page. 

image
image

### CODE REVIEW

File: Pages/Authentication/login.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            <a href="../../index.html" class="link"><p class="header-heading">AMAZE-ON</p></a>
+        </div>
+        <div class="header-right-icons">
+            <a href="../Authentication/signup.html"><button class="btn btn-primary-solid header-btn">Signup</button></a>
+        </div>
+    </header>
+
+    <section class="login-container">
+        <div class="card">
+            <!-- <div class="card-wrapper"> -->
```
Comment (tushargoyal812) [inline Pages/Authentication/login.html]: remove comments

File: Pages/Authentication/signup.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            <a href="../../index.html" class="link"><p class="header-heading">AMAZE-ON</p></a>
+        </div>
+        <div class="header-right-icons">
+            <a href="../Authentication/login.html"><button class="btn btn-primary-solid header-login-btn">Login</button></a>
+        </div>
+    </header>
+
+    <section class="login-container">
+        <div class="card">
+            <!-- <div class="card-wrapper"> -->
```
Comment (tushargoyal812) [inline Pages/Authentication/signup.html]: also here

**Review by tushargoyal812 — COMMENTED** [review]
looks good to merge

File: Pages/Authentication/login.html [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css"
+    integrity="sha512-Fo3rlrZj/k7ujTnHg4CGR2D7kSs0v4LLanw2qksYuRlEzO+tcaEPQogQ0KaoGN26/zrn20ImR1DfuLWnOo7aBA=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+<link href="https://fonts.googleapis.com/icon?family=Material+Icons+Outlined" rel="stylesheet">
+<link rel="stylesheet" href="https://amaze-ui.netlify.app/css/components.css">
+<link rel="stylesheet" href="https://amaze-ui.netlify.app/css/main.css">
+<link rel="stylesheet" href="https://amaze-ui.netlify.app/css/header.css">
+<link rel="stylesheet" href="../../css/header.css">
+<link rel="stylesheet" href="login.css">
+    <title>Login</title>
```
Comment (syed-shaida-hussain) [inline Pages/Authentication/login.html:17]: Run prettier on this file

**Review by syed-shaida-hussain — APPROVED** [review]
Everything looks good. You can merge it

File: Pages/Authentication/login.html (hunk 1 above)
Comment (sharmayatendra) [reply Pages/Authentication/login.html]: done

File: Pages/Authentication/login.html (hunk 3 above)
Comment (sharmayatendra) [reply Pages/Authentication/login.html:17]: done

**Pull request merged**